src/views/forms/AccountManagementForms/DepositForm.tsx

```diff
@@ -105,6 +107,16 @@ export const DepositForm = ({ onDeposit, onError }: DepositFormProps) => {
   const debouncedAmountBN = MustBigNumber(debouncedAmount);
   const balanceBN = MustBigNumber(balance);
 
+  useEffect(() => {
```

Comment:
logging to bugsnag could make sense here to track what traders are seeing as error message, but i wonder if we should move the amplitude tracking to abacus since it seems like there can be multiple errors(?) + share logic with mobile?

Reply:
hm... i was under the impression that amplitude wants to track the user journey - in that sense it's more accurate to only track the error that the user sees. 

whereas in bugsnag i feel like we'd want to track all the errors since it's more of an error tracking/triaging platform?

i'd be more tempted to log errors on the abacus level if that makes sense